### PR TITLE
Migrate to new reporter.

### PR DESCRIPTION
**Pull request by dipak-pawar**
#### Short description of what this resolves:
Added new reporter integration

#### Changes proposed in this pull request:

- Added new reporter integration
- Updated mvn dependencies to use latest stable versions.

Note: this build is depends on cube SNAPSHOT dependency. 

**Fixes**: #25

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -52,19 +52,18 @@
...
+        <version.arquillian_cube>1.0.0.Alpha20</version.arquillian_cube>
         <version.toxiproxyjava>2.1</version.toxiproxyjava>
-        <version.junit>4.11</version.junit>
-        <version.mockito>1.10.19</version.mockito>
-        <version.assertj>2.4.1</version.assertj>
-        <version.arquillian.recorder>1.1.5.Final</version.arquillian.recorder>
+        <version.junit>4.12</version.junit>
+        <version.mockito>2.7.17</version.mockito>
+        <version.assertj>3.6.2</version.assertj>
+        <version.arquillian.reporter>0.0.2</version.arquillian.reporter>
```
Comment (MatousJobanek) [inline pom.xml]: please add to this commit only relevant changes - updates of junit or arquillian-core versions put into a separate commit

File: reporter/src/main/java/org/arquillian/cube/q/recorder/TakeNetworkChaosInformation.java [hunk 2]
```diff
@@ -80,32 +80,32 @@ public void addToxicInfoToToxics(QNetworkChaosToxic.ToxicAction toxicAction, Tox
...
     public void reportToxicConfiguration(@Observes After event, ReporterConfiguration reporterConfiguration) throws IOException {
 
-        final String fileName = event.getTestMethod().getName() + ".json";
+        final Method testMethod = event.getTestMethod();
+        final String testMethodName = testMethod.getName();
+        final String fileName = testMethodName + ".json";
 
         final FileEntry fileEntry = createFileEntryWithJSON(reporterConfiguration, fileName);
+        Reporter.createReport(new TestMethodReport(testMethodName))
+                .addEntry(fileEntry)
```
Comment (MatousJobanek) [inline reporter/src/main/java/org/arquillian/cube/q/recorder/TakeNetworkChaosInformation.java]: at this point, I would expect addKeyValueEntry where the key is some kind of description what the file contains etc... As a value you can use any Entry, so also FileEntry

**Review by MatousJobanek — CHANGES_REQUESTED** [review]

**Comment by bartoszmajsak** [comment]
- [x] Resolve merge conflicts
- [x] Update Cube

File: pom.xml [hunk 3]
```diff
@@ -53,18 +53,17 @@
...
-        <version.arquillian_cube>1.0.0.Alpha15</version.arquillian_cube>
+        <version.arquillian_cube>1.0.0.Alpha20</version.arquillian_cube>
         <version.toxiproxyjava>2.1.1</version.toxiproxyjava>
         <version.junit>4.12</version.junit>
-        <version.mockito>1.10.19</version.mockito>
-        <version.assertj>2.6.0</version.assertj>
-        <version.arquillian.recorder>1.1.6.Final</version.arquillian.recorder>
+        <version.mockito>2.7.17</version.mockito>
+        <version.assertj>3.6.2</version.assertj>
+        <version.arquillian.reporter>0.0.2</version.arquillian.reporter>
```
Comment (bartoszmajsak) [inline pom.xml]: I believe there is `0.0.3` out in the wild already.

**Review by bartoszmajsak — APPROVED** [review]
Just one small thing. Other than that LGTM

**Review by lordofthejars — APPROVED** [review]

**Comment by MatousJobanek** [comment]
How about the versions in the main pom - shouldn't be the version updates part of another commit/PR? Or do we somehow depend on the updates in this PR?

**Comment by dipak-pawar** [comment]
@MatousJobanek We depends on new version of assertJ & Cube for this PR. I don't want to create new Commit or PR only to change mockito's version

**Comment by MatousJobanek** [comment]
OK, cool. Then LGTM :-)
Thanks @dipak-pawar

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**